requirements/build.windows.txt

```diff
@@ -1,64 +1,64 @@
-#
-# This file is autogenerated by pip-compile
-# To update, run:
-#
-#    python boots.py lock
-#
--e git+https://github.com/altendky/pysunspec@3662dd0a7d6810652320284ac4256beee5b20f2d#egg=pysunspec
-    # via -r requirements\base.in
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    python boots.py lock
+#
 altgraph==0.17
```

Comment:
There are a lot of changes to the build.windows.txt file. I assume these are line ending differences. I'm not saying this is wrong, I'm just verifying that I am using the correct setting for git commits. I am using the git setting `core.autocrlf=true`. Is that correct?

Reply:
Hmm...  I generally set Windows git to checkout and commit as-is when I install it.  I guess that would be `=false` probably.  I think Todd had it set to change to `lf` on commit.

It does appear that I committed with `crlf`.  While I guess this is 'correct' based on history, it isn't what I actually want it to be.

![image](https://user-images.githubusercontent.com/543719/108531691-f8d76200-72a4-11eb-8a74-1fa5ef318c3f.png)

While we may want to define some proper setting for Git also, I think the correct solution here is for `boots` to normalize to `lf`.  I guess it will have to stop using `.extractall()` and rather get the bytes of each file and write them individually to gain control over the line endings.

https://github.com/altendky/boots/blob/9d399eed1fb7448dd3be7e707a787202f1755918/boots.py#L691-L692
```python
        with tarfile.open(artifact_path, mode='r:gz') as tar:
            tar.extractall(path=configuration.project_root)
```

I'll fix this manually (`dos2unix`) to reduce the diff and update here.  `boots` can be fixed separately in https://github.com/altendky/boots/issues/4.